src/ODE.jl

```diff
@@ -8,9 +8,9 @@ using Compat
 
 ## minimal function export list
 # adaptive non-stiff:
-export ode23, ode45, ode78
+export ode23, ode45, ode78, ode113
 # non-adaptive non-stiff:
-export ode4, ode4ms
+export ode4, ode4ms, ode_am4
```

Comment:
I changed to naming to `ode4am`, which fits with conventions better. And yeah, I think we were planning to open the PR against the original repo, and save the JuliaODE fork for work on iterative solvers.